lib/tree_reject.rb

```diff
@@ -55,4 +55,4 @@ def extract_subtrees(ignored_keys, ignored_leaves)
   end
 end
 
-Hash.send(:include, TreeReject::TreeRejectHash)
+Hash.send(:include, TreeReject::TreeRejectHash) # rubocop:disable Lint/SendWithMixinArgument
```

Comment:
`include` is public so the `send` isn't required.

```suggestion
Hash.include(TreeReject::TreeRejectHash)
```